packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts

```diff
@@ -231,64 +277,96 @@ export class PushNotificationsService {
   }
 
   _checkShouldSendNotif(notification, cb) {
-    if (notification.type != 'NewTxProposal') return cb(null, true);
+    if (notification.type != 'NewTxProposal' || notification.type == 'NewBlock') return cb(null, true);
     this.storage.fetchWallet(notification.walletId, (err, wallet) => {
       return cb(err, wallet && wallet.m > 1);
     });
   }
 
   _getRecipientsList(notification, notificationType, cb) {
-    this.storage.fetchWallet(notification.walletId, (err, wallet) => {
-      if (err) return cb(err);
-
-      let unit;
-      if (wallet.coin != Defaults.COIN) {
-        unit = wallet.coin;
-      }
+    if (notification.type === 'NewBlock') {
+      this.storage.fetchLatestPushNotificationSubs((err, subs) => {
+        if (err) return cb(err);
 
-      this.storage.fetchPreferences(notification.walletId, null, (err, preferences) => {
-        if (err) logger.error(err);
-        if (_.isEmpty(preferences)) preferences = [];
+        async.map(
```

Comment:
Add a log.info before this: "Sending new block [coin/chain] notifications to: xxx subscribers"